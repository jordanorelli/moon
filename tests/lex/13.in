# https://github.com/jordanorelli/moon/issues/2
some_path: /path/to/file
