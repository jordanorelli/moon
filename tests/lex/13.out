{t_comment  https://github.com/jordanorelli/moon/issues/2}
{t_name some_path}
{t_object_separator :}
{t_string /path/to/file}
